evaluate.py

```diff
@@ -1622,7 +1628,7 @@ def _benchmark_vm(
         if validator:
             if isinstance(validator, Validator):
                 ref_outputs = validator.GetReference()
-                validator.Validate(vm, ref_outputs)
+                validator.Validate(vm, ref_outputs, data)
```

Comment:
How does it work? It looks like method `Validate` takes only 2 arguments but you pass 3...

Reply:
For some reason the corresponding change in `Validate` was not in commit. Should be fine now.